constrained-voucher.mkd

```diff
@@ -646,25 +646,37 @@ signed with a COSE_Sign1 structure {{I-D.ietf-cose-rfc8152bis-struct}}.
     Subtype name:  voucher-cose+cbor
     Required parameters:  none
     Optional parameters:  none
-    Encoding considerations:  binary
+    Encoding considerations:  binary (CBOR)
     Security considerations:  Security Considerations of THIS RFC.
     Interoperability considerations:  The format is designed to be
       broadly interoperable.
     Published specification:  THIS RFC.
     Applications that use this media type:  ANIMA, 6tisch, and other
       zero-touch onboarding systems
+   Fragment identifier considerations:  The syntax and semantics of
+      fragment identifiers specified for application/voucher-cose+cbor are
+      as specified for application/cbor.  (At publication of this
+      document, there is no fragment identification syntax defined for
+      application/cbor.)
     Additional information:
       Magic number(s):  None
       File extension(s):  .vch
       Macintosh file type code(s):  none
     Person & email address to contact for further information:  IETF
-      ANIMA WG
-    Intended usage:  LIMITED
+      ANIMA WG [^ouch]
+    Intended usage:  LIMITED [^ouch2]
     Restrictions on usage:  NONE
     Author:  ANIMA WG
     Change controller:  IETF
     Provisional registration? (standards tree only):  NO
 
+[^ouch]: That will need to be fixed so there is a contact point beyond
+    the time the ANIMA WG exists.  Please see RFC 8710 for an example.
+
```

Comment:
so, please either tell me what to put, or raise an issue.
A pull request polluting the text with a pointer to something to do is rather non-actionable.

Reply:
ART has defined rules for that; I could try to recollect those.  But this is a different area, so I'd ask the AD.

Reply:
See also I-D-to-be https://cabo.github.io/cbor-media-types/draft-bormann-cbor-defining-media-types.html